src/main/java/reactor/netty/ByteBufFlux.java

```diff
@@ -84,10 +84,14 @@ public static ByteBufFlux fromString(Publisher<? extends String> source) {
 
 	public static ByteBufFlux fromString(Publisher<? extends String> source, Charset charset, ByteBufAllocator allocator) {
 		Objects.requireNonNull(allocator, "allocator");
-		return new ByteBufFlux(Flux.from(source)
-		                           .map(s -> allocator.buffer()
-		                                              .writeBytes(s.getBytes(charset))),
-		                       allocator);
+		return new ByteBufFlux(
+				Flux.from(source)
+				    .map(s -> {
+				        ByteBuf buffer = allocator.buffer();
```

Comment:
I wonder if we need an allocator at all (or by default it should be unpooled) since we wrap bytes. I wonder about `Unpooled.wrappedBytes()`